source/declarations.tex

```diff
@@ -4370,6 +4370,31 @@
 is called
 \defn{direct-initialization}.
 
+\pnum
+An \defnadj{implicitly movable}{entity} is
+a variable with automatic storage duration
+that is either a non-volatile object or
+an rvalue reference to a non-volatile object type.
+The copy-initialization performed for one of the following is
+known as \defn{result-initialization}:
```

Comment:
Surely it's the initialization performed _from_ one of these, right? It's not the initialization _of_ the operand of the `return` statement; it's the initialization of the returned object, which is copy-initialized _from_ the operand of the `return`.

Same comment down where you talk about a `co_return` statement "whose operand is _subject to_ result-initialization." The operand is actually _used_ to result-initialize whatever is passed to `return_value`.